casper/src/main/scala/coop/rchain/casper/merging/DagMerger.scala

```diff
@@ -47,28 +47,24 @@ object DagMerger {
             as.map(_.eventLogIndex).toList.combineAll,
             bs.map(_.eventLogIndex).toList.combineAll
           )
-
-      computeTrieActions = (changes: StateChange, mergeableChs: NumberChannelsDiff) => {
-        for {
-          historyReader <- historyRepository.getHistoryReader(lfbPostState)
-          baseReader    = historyReader.readerBinary
-          baseGetData   = baseReader.getData(_: Blake2b256Hash).map(_.map(_.decoded))
-          overrideTrieAction = (
-              hash: Blake2b256Hash,
-              changes: ChannelChange[ByteVector],
-              numberChs: NumberChannelsDiff
-          ) =>
-            numberChs.get(hash).traverse {
-              RholangMergingLogic.calculateNumberChannelMerge(hash, _, changes, baseGetData)
-            }
-          actions <- StateChangeMerger.computeTrieActions(
-                      changes,
-                      baseReader,
-                      mergeableChs,
-                      overrideTrieAction
-                    )
-        } yield actions
-      }
+      historyReader <- historyRepository.getHistoryReader(lfbPostState)
+      baseReader    = historyReader.readerBinary
+      baseGetData   = baseReader.getData(_: Blake2b256Hash).map(_.map(_.decoded))
```

Comment:
To simplify my own code. Without binary part getData can be used from historyReader directly.
`baseGetData  = historyReader.getData _`